src/components/Header/header.module.css

```diff
@@ -0,0 +1,25 @@
+:root {
```

Comment:
put variables in varieable.css file, make variable CSS file in styles folder

Reply:
creat new global variable.css file?